src/imaCore.mjs

```diff
@@ -759,58 +605,38 @@ async function callbackAllMessagesSign( optsTransfer, err, jarrMessages, joGlueR
-                        cc.notice( optsTransfer.joMessageProxyDst.address ) +
-                        cc.debug( " contract..." ) + "\n" );
-                }
+                optsTransfer.details.debug( "{p}Verifying the {} event of the MessageProxy/{} " +
+                    "contract...", optsTransfer.strLogPrefix, strEventName,
+                optsTransfer.joMessageProxyDst.address );
                 const joEvents = await imaEventLogScan.getContractCallEvents(
                     optsTransfer.details, optsTransfer.strLogPrefixShort,
-                    optsTransfer.ethersProviderDst,
-                    optsTransfer.joMessageProxyDst, strEventName,
+                    optsTransfer.ethersProviderDst, optsTransfer.joMessageProxyDst, strEventName,
                     joReceipt.blockNumber, joReceipt.transactionHash,
                     optsTransfer.joMessageProxyDst.filters[strEventName]() );
                 if( joEvents.length == 0 ) {
-                    if( log.verboseGet() >= log.verboseReversed().debug ) {
-                        optsTransfer.details.write( optsTransfer.strLogPrefix +
-                            cc.success( "Success, verified the " ) + cc.info( strEventName ) +
-                            cc.success( " event of the " ) + cc.info( "MessageProxy" ) +
-                            cc.success( "/" ) +
-                            cc.notice( optsTransfer.joMessageProxyDst.address ) +
-                            cc.success( " contract, no events found" ) + "\n" );
-                    }
+                    optsTransfer.details.success( "{p}Success, verified the {} event of the " +
+                        "MessageProxy/{} contract, no events found", optsTransfer.strLogPrefix,
+                    strEventName, optsTransfer.joMessageProxyDst.address );
                 } else {
-                    if( log.verboseGet() >= log.verboseReversed().critical ) {
-                        const strError = optsTransfer.strLogPrefix +
-                            cc.fatal( "CRITICAL ERROR:" ) + cc.warning( " Failed" ) +
-                            cc.error( " verification of the " ) +
-                            cc.warning( "PostMessageError" ) + cc.error( " event of the " ) +
-                            cc.warning( "MessageProxy" ) + cc.error( "/" ) +
-                            cc.notice( optsTransfer.joMessageProxyDst.address ) +
-                            cc.error( " contract, found event(s): " ) + cc.j( joEvents ) + "\n";
-                        optsTransfer.details.write( strError );
-                        if( log.id != optsTransfer.details.id )
-                            log.write( strError );
+                    optsTransfer.details.critical( "{p}Failed verification of the {} event of " +
+                        "the {}/{} contract, found event(s): {}", optsTransfer.strLogPrefix,
+                    "PostMessageError", "MessageProxy", optsTransfer.joMessageProxyDst.address,
```

Comment:
You can move "PostMessageError", "MessageProxy" into log string

Reply:
Done